Project.toml

```diff
@@ -4,15 +4,14 @@ repo = "https://github.com/JuliaDynamics/InteractiveDynamics.jl.git"
 version = "0.14.6"
 
 [deps]
-AbstractPlotting = "537997a7-5e4e-5d89-9595-2241ea00577e"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
 Observables = "510215fc-4207-5dde-b226-833fc4488ee2"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-AbstractPlotting = "0.15, 0.16, 0.17, 0.18"
 DataStructures = "0.17, 0.18"
```

Comment:
We need to add the latest makie version here

Reply:
Done!